src/js/serial.js

```diff
@@ -1,51 +1,41 @@
 'use strict';
 
-var serial = {
-    connected:       false,
-    connectionId:    false,
-    openRequested:   false,
-    openCanceled:    false,
-    bitrate:         0,
-    bytesReceived:   0,
-    bytesSent:       0,
-    failed:          0,
-    connectionType:  'serial', // 'serial' or 'tcp'
-    connectionIP:    '127.0.0.1',
-    connectionPort:  2323,
+const serial = {
+    connected:      false,
+    connectionId:   false,
+    openCanceled:   false,
+    bitrate:        0,
+    bytesReceived:  0,
+    bytesSent:      0,
+    failed:         0,
+    connectionType: 'serial', // 'serial' or 'tcp'
+    connectionIP:   '127.0.0.1',
+    connectionPort: 5761,
 
     transmitting:   false,
-    outputBuffer:  [],
-
-    logHead: 'SERIAL: ',
+    outputBuffer:   [],
 
     connect: function (path, options, callback) {
-        var self = this;
-        var testUrl = path.match(/^tcp:\/\/([A-Za-z0-9\.-]+)(?:\:(\d+))?$/)
+        const self = this;
+        const testUrl = path.match(/^tcp:\/\/([A-Za-z0-9\.-]+)(?:\:(\d+))?$/);
         if (testUrl) {
             self.connectTcp(testUrl[1], testUrl[2], options, callback);
         } else {
             self.connectSerial(path, options, callback);
         }
     },
     connectSerial: function (path, options, callback) {
-        var self = this;
-        self.openRequested = true;
+        const self = this;
         self.connectionType = 'serial';
-        self.logHead = 'SERIAL: ';
 
         chrome.serial.connect(path, options, function (connectionInfo) {
-            if (chrome.runtime.lastError) {
```

Comment:
Not sure if removing error logging is going in the right direction.

Reply:
See checkChromeRunTimeError() on next line

Reply:
:-)